noty-android/app/simpleapp/src/main/java/dev/shreyaspatil/noty/simpleapp/view/detail/NoteDetailFragment.kt

```diff
@@ -116,6 +116,14 @@ class NoteDetailFragment :
         if (errorMessage != null) {
             toast("Error: $errorMessage")
         }
+
+        updatePinnedIcon(state.isPinned)
+    }
+
+    private fun updatePinnedIcon(isPinned: Boolean) {
+        pinMenuItem?.run {
+            if (isPinned) setIcon(R.drawable.ic_pinned) else setIcon(R.drawable.ic_unpinned)
+        }
```

Comment:
This code can be simplified and written better as
```suggestion
        pinMenuItem?.run {
            val icon = if (isPinned) R.drawable.ic_pinned else R.drawable.ic_unpinned
            setIcon(icon)
        }
```

I feel this much readable, WDYT?

Reply:
Yes!